dali/kernels/common/scatter_gather.h

```diff
@@ -77,16 +77,23 @@ class DLL_PUBLIC ScatterGatherGPU {
     }
   }
 
+  enum class CopyMethod {
+    Default = 0,  // Uses scatter-gather kernel, unless there are 2 or fewer single effective copy
+                  // ranges, in that cases cudaMemcpyAsync is used
+    MemcpyAlways = 1,  // Always use cudaMemcpyAsync
+    MemcpyNever = 2,   // Always use scatter-gather kernel
+  };
+
   /**
    * @brief Executes the copies
-   * @param stream - the cudaStream on which the copies are scheduled
-   * @param reset - if true, calls Reset after processing is over
-   * @param useMemcpyOnly - if true, all copies are executed using cudaMemcpy;
-   *                        otherwise a batched kernel is used if there are more than 2 ranges
-   * @param memcpyKind    - determines the cudaMemcpyKind when using cudaMemcpy
+   * @param stream     - the cudaStream on which the copies are scheduled
+   * @param reset      - if true, calls Reset after processing is over
```

Comment:
Maybe it would be good to describe, what `Reset` does? Probably in `Reset` function docs. Just food for thought

Reply:
Done